src/locales/en.json

```
@@ -220,7 +222,8 @@
     "placeImage": "Click to place the image, or click and drag to set its size manually",
     "publishLibrary": "Publish your own library",
     "bindTextToElement": "Press enter to add text",
-    "deepBoxSelect": "Hold CtrlOrCmd to deep select, and to prevent dragging"
+    "deepBoxSelect": "Hold CtrlOrCmd to deep select, and to prevent dragging",
+    "eraserDeselect": "Hold Alt while holding the mouse over to deselect the elements for removal"
```

```suggestion
    "eraserDeselect": "Hold Alt un-mark elements for removal"
```

TBH I don't know how to word this one, but the current tooltip should be improved.